<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>app_widget</class>
 <widget class="QWidget" name="app_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>315</width>
     <height>349</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="app_layout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLCDNumber" name="timer_lcd">
      <property name="minimumSize">
       <size>
        <width>271</width>
        <height>73</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLCDNumber{
	color:rgb(255, 255, 255);
	background-color:rgb(0, 0, 0);
	border-radius: 5px;
}</string>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>12345678.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="play_pause_btn">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>play-button.png</normaloff>play-button.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop_btn">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>stop-button.png</normaloff>stop-button.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>微軟正黑體</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>H</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QComboBox" name="hour_combo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>微軟正黑體</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>M</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QComboBox" name="minute_combo">
            <property name="maxCount">
             <number>60</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>微軟正黑體</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>S</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QComboBox" name="second_combo"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="forever_radio">
          <property name="text">
           <string>forever</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QRadioButton" name="repeat_radio">
          <property name="text">
           <string>repeat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="repeat_combo">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="hint_line">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>hint text</string>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::LogicalMoveStyle</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="hide_btn">
          <property name="text">
           <string>running backgroud</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exit_btn">
          <property name="styleSheet">
           <string notr="true">QPushButton:checked {color: white;}</string>
          </property>
          <property name="text">
           <string>close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
